hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md

```diff
@@ -306,20 +306,22 @@ The capacity scheduler supports hierarchical queues. This one request will print
 | Item | Data Type | Description |
 |:---- |:---- |:---- |
 | type | string | Scheduler type - capacityScheduler |
-| capacity | float | Configured queue capacity in percentage relative to its parent queue |
+| capacity | float | **Configured** queue capacity in percentage relative to its parent queue in legacy-queue-mode, otherwise the **effective** capacity  |
```

Comment:
Is it OK to use the `effective` term here? The json/xml responses are full of effective* objects anyway.

@brumi1024

Reply:
Yes, here I don't see an issue with it. If there are no other mentions of this maybe a short, few word explanation can help, mentioning that it might be already downscaled to fit in the parent's resources etc.